debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/transforms/partitions/ComputePartition.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql.transforms.partitions;
```

Comment:
There is no need to have this SMT MySQL specific. Please move it to `debezium-core` side-by-side with other SMTs.